processing/src/test/java/io/druid/query/groupby/GroupByTimeseriesQueryRunnerTest.java

```diff
@@ -112,6 +133,45 @@ public GroupByTimeseriesQueryRunnerTest(QueryRunner runner)
     super(runner, false);
   }
 
+  // GroupBy handles timestamps differently when granularity is ALL
```

Comment:
GroupBy v1 and v2 behave the same in this case, when the query granularity is set to all, the result timestamps will contain the earliest timestamp in the query interval, which differs from Timeseries

Prior to this patch, `testFullOnTimeseriesMaxMin()` was not executing `mergeResults()` on GroupBy V1, so the timestamp wasn't truncated to the ALL granularity, so the results matched the Timeseries results exactly.